--- libmimedir-0.5.1/Makefile.in.orig	2015-01-31 09:06:10.851265831 +0100
+++ libmimedir-0.5.1/Makefile.in	2015-01-31 09:49:19.547823861 +0100
@@ -15,23 +15,23 @@
 INSTALL= @INSTALL@
 RANLIB= @RANLIB@
 LIBTOOL= @LIBTOOL@
-LIBS= -L. libmimedir.la
+LIBS= libmimedir-vlm.la
 
 OBJS= dirlex.o dirsynt.o parse.o get.o dup.o free.o memmem.o add.o count.o del.o
 LOBJS= dirlex.lo dirsynt.lo parse.lo get.lo dup.lo free.lo memmem.lo add.lo count.lo del.lo
 
 TESTS = test
 
-all: libmimedir.a libmimedir.la $(TESTS)
+all: libmimedir-vlm.la $(TESTS)
 
-libmimedir.a: ${OBJS}
-	ar rc libmimedir.a ${OBJS}
-	$(RANLIB) libmimedir.a
+libmimedir-vlm.a: ${OBJS}
+	ar rc libmimedir-vlm.a ${OBJS}
+	$(RANLIB) libmimedir-vlm.a
 
-libmimedir.la: ${LOBJS}
-	$(LIBTOOL) --mode=link ${CC} ${CFLAGS} ${LDFLAGS} -rpath ${libdir} -o libmimedir.la ${LOBJS}
+libmimedir-vlm.la: ${LOBJS}
+	$(LIBTOOL) --mode=link ${CC} ${CFLAGS} ${LDFLAGS} -rpath ${libdir} -o libmimedir-vlm.la ${LOBJS}
 
-test: test.o libmimedir.la
+test: test.o libmimedir-vlm.la
 	$(LIBTOOL) --mode=link ${CC} ${CFLAGS} ${LDFLAGS} -o test test.o ${LIBS}
 
 dirsynt.c dirsynt.h: dirsynt.y
@@ -55,9 +55,9 @@
 	./test
 
 install: all
-	${INSTALL} -m 444 libmimedir.h ${includedir}
-	${INSTALL} -m 444 libmimedir.a ${libdir}
-	$(LIBTOOL) --mode=install ${INSTALL} -c -m 444 libmimedir.la ${libdir}/libmimedir.la
+	$(INSTALL) -d $(DESTDIR)$(includedir) $(DESTDIR)$(libdir)
+	${INSTALL} libmimedir.h $(DESTDIR)${includedir}
+	$(LIBTOOL) --mode=install ${INSTALL} -c libmimedir-vlm.la $(DESTDIR)${libdir}/libmimedir-vlm.la
 
 clean:
 	rm -f $(OBJS) $(LOBJS) test.o *.a *.la *.core
